images/macos/helpers/Xcode.Helpers.psm1

```diff
@@ -121,4 +121,29 @@ function Get-XcodePairsList {
         $result += "$watchName $phoneName"
     }
     return $result
+}
+
+function Invoke-XCVersion {
+    param(
+        [Parameter(Mandatory)]
+        [string] $Command,
+        [Parameter()]
+        [int] $RetryAttempts = 7,
+        [Parameter()]
+        [int] $PauseDuration = 1
+    )
+
+    do {
+        Write-Host "Current attempt: [$RetryAttempts]"
+        $result = Get-CommandResult -Command $Command -Multiline
+        Write-Host "Exit code: [$($result.ExitCode)]"
+        Write-Host "Output message: "
+        $($result.Output)
```

Comment:
Should we use `Write-Host` here

Reply:
changed to `$result.Output | ForEach-Object { Write-Host $_ }`